notebooks/amplicon_mapping_file_generator.ipynb

```diff
@@ -333,7 +354,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "merged_output_filename = 'test_output/amplicon/20230203_IL515fBC_806_ABTX_11052_Plates_174_178_182_185_204_207_210_215_.txt'"
+    "# additional prep-prep files to merge w/qiita_df\n",
+    "files = ['./20230201_IL515fBC_806r_ABTX_11052_174_178_182_185_MF_notebook.txt']\n",
+    "\n",
+    "# (the mapping-file output above can be combined w/itself for testing purposes)\n",
```

Comment:
should this be pre-preparation?

Reply:
It could! Do you think we should go ahead and rename it to amplicon_pre_prep_file_generator.ipynb as well?

Reply:
yes!